### PR TITLE
Improve show role

### PR DESCRIPTION
**Pull request by yp05327**
Add a general show role template.

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Review by wxiaoguang — DISMISSED** [review]
Please use new `ctx.Locale` instead of `"locale" $.locale`

**Comment by yp05327** [comment]
> Please use new `ctx.Locale` instead of `"locale" $.locale`

Done.

File: templates/repo/issue/view_content.tmpl [hunk 1]
```diff
@@ -44,18 +44,7 @@
...
 						</div>
 						<div class="comment-header-right actions gt-df gt-ac">
-							{{if gt .Issue.ShowRole 0}}
-								{{if (.Issue.ShowRole.HasRole "Writer")}}
-									<div class="ui basic label role-label">
-										{{$.locale.Tr "repo.issues.collaborator"}}
-									</div>
-								{{end}}
-								{{if (.Issue.ShowRole.HasRole "Owner")}}
-									<div class="ui basic label role-label">
```
Comment (wxiaoguang) [inline templates/repo/issue/view_content.tmpl:54]: There were "role-label" classes, they are lost now?

Comment (yp05327) [reply templates/repo/issue/view_content.tmpl:54]: Only in this place have `role-label` class, didn't understand the usage here, but I understand the usage of `role-label` now: hide them in mobiles.

In GitHub will not display role label in mobiles, so we can add this in the general template.
image
image

In Gitea:
image
After:
image

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**